phpstan.neon.dist

```diff
@@ -8,3 +8,8 @@ parameters:
         - tests
     bootstrapFiles:
         - tests/bootstrap.php
+
+    ignoreErrors:
+        # Ignore errors about classes removed from doctrine/dbal 3.x
+        - '/Class Doctrine\\DBAL\\Platforms\\(MySqlPlatform|PostgreSqlPlatform) not found\./'
+        - '/Instantiated class Doctrine\\DBAL\\Driver\\PDOException not found\./'
```

Comment:
`MySqlPlatform` had its casing changed to `MySQLPlatform` in DBAL 3.  That one's pretty much a non-issue with PHP being case insensitive with class names, but PHPStan flags it (could probably fix it to the updated casing and be done with it).  The other two classes are fully removed in DBAL 3.  Unless the PHPStan config is going to get crazy and a run is done with both DBAL 2 and DBAL 3 (some of the Doctrine repos are doing this), just ignoring these would be the best way to deal with it.  I didn't put it in the baseline just because I did this commit through the GitHub UI and don't have the time right now to do a full clone, run PHPStan, and re-generate the baseline file, but it's easy to move these ignores if need be.

Reply:
If these classes are not present in doctrine / dbal 3.x and our code is relying on them without any BC layer, I think we can't allow the version bumping until this situation is resolved.

Reply:
Of the 6 reported errors, 4 of them come from the class casing and 2 are B/C conditionals.  PHPStan just can't work that out on its own.

The 3 class not found errors from `TranslationWalker.php` are a mix of the mixed `MySqlPlatform` versus `MySQLPlatform` casing or the B/C layer applied in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2284/files#diff-07d23b78d99c87e3121e586d76ea31fb9f3a5f2eb97602a123c6811074f5d379R464-R468

The class not found error from `CustomerType.php` has the B/C layer at https://github.com/doctrine-extensions/DoctrineExtensions/pull/2284/files#diff-367f5ccd73a06127b2dcc52d40484b3f73bb19fd10ac5c1e93266f04f2ab68e5L91-L94 covering it

And the 2 class not found errors from `BaseTestCaseOM.php` and `BaseTestCaseORM.php` are again the mixed MySQL platform cases.